src/rdkafka_sasl_oauthbearer_oidc.c

```diff
@@ -351,13 +351,13 @@ void rd_kafka_oidc_token_refresh_cb(rd_kafka_t *rk,
         }
 
         if (rk->rk_conf.sasl.oauthbearer.extensions_str) {
-                extensions = rd_string_split(
```

Comment:
I think these style changes have been fixed on master, so next time you rebase you can remove this commit.